projects/angular/src/forms/datepicker/date-input.validator.ts

```diff
@@ -33,3 +34,57 @@ export class ClrDateInputValidator implements Validator {
     return null;
   }
 }
+
+@Directive({
+  selector: '[clrRangeStartDate]',
```

Comment:
```suggestion
  selector: '[clrStartDateInput]',
```

Same as above, I think "range" is redundant here. Also, it should end in "input" like `clrDateInput`. Please rename the class and the end date directive as well.